flu_ar2/README.md

```diff
@@ -3,11 +3,13 @@
 To test this out locally, run the following with `flu_ar2` as your working directory.  Note that somewhere along the line, incompatible versions of pandas and numpy ended up in a requirements file somewhere.  I have not dug into where or why that happened.  Here we manually override the pandas installation.
 
 ```
+cd <this_repo>
```

Comment:
noting that instructions above say to run all of the below with `flu_ar2` as the working directory.  the idea i had with this was that we may want to allow for a separate set of requirements per model.  in practice, though, for now the requirements will be largely the same across models...